src/Canvas.js

```diff
@@ -24,6 +24,27 @@ class Canvacord {
     constructor() {
         throw new Error(`The class ${this.constructor.name} may not be instantiated!`);
     }
+    
+    /**
+     * colorify
+     * @param {Image} image to color
+     * @param {Hex} hex to color the image
+     * @returns {Promise<Buffer>}
+     * @example let img = await canva.colorify(img, hex);
+     * canva.write(img, "img.png");
+     */
+    static async colorify(image1, hex) {
```

Comment:
Looks OK , but one question the function takes only one `Image` so shouldn't the `image1` parameter should only be `image` and also here - 
```js
 if (!image1) throw new Error("first image was not provided!");
```
Using `first image` doesn't seem OK

Reply:
Well, yeah lmao i didn't noticed it!